src/pages/Environments.js

```diff
@@ -417,39 +420,44 @@ const Environments = () => {
         rowCount: filteredDisks.length,
         columns: [
           {
-            field: 'project',
-            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'project', onSort: setDiskSort }, ['Billing project']),
+            field: 'billing-project',
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'billing-project', onSort: setDiskSort }, ['Billing project']),
             cellRenderer: ({ rowIndex }) => {
-              const { status: diskStatus, googleProject } = filteredDisks[rowIndex]
+              const { status: diskStatus, googleProject, labels: { saturnWorkspaceNamespace } } = filteredDisks[rowIndex]
               const appType = getDiskAppType(filteredDisks[rowIndex])
               const multipleDisksOfType = _.remove(disk => getDiskAppType(disk) !== appType || disk.status === 'Deleting',
                 disksByProject[googleProject]).length > 1
-              const forAppText = !!appType ? ` for ${_.capitalize(appType)}` : ''
+              const forAppText = !!appType ? `for ${_.capitalize(appType)}` : ''
               return h(Fragment, [
-                googleProject,
+                saturnWorkspaceNamespace,
                 diskStatus !== 'Deleting' && multipleDisksOfType &&
                 h(TooltipTrigger, {
-                  content: `This billing project has multiple active persistent disks${forAppText}. Only the latest one will be used.`
+                  content: `This billing project has multiple active persistent disks ${forAppText}. Only the latest one will be used.`
                 }, [icon('warning-standard', { style: { marginLeft: '0.25rem', color: colors.warning() } })])
               ])
             }
           },
+          {
+            field: 'workspace',
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'workspace', onSort: setDiskSort }, ['Workspace']),
```

Comment:
The sorting isn't working correctly for me on this page. I think the problem for "Type" may be that the sorting is only being done within apps and runtimes, which is very unexpected. And for Workspace, it doesn't seem to be doing anything at all.

![image](https://user-images.githubusercontent.com/484484/148785171-974d23cf-0df6-48e1-894f-e313eeb9691b.png)

![image](https://user-images.githubusercontent.com/484484/148785257-64360e39-e0c6-494d-a332-788bcbdc2a6d.png)

Reply:
Ugh yea…we seem to have accumulated regressions over time. I've added sorting for the new `Workspace` column, but would prefer to file a bug to address the pre-existing issues properly (with unit testing if possible) since sorting is no worse as part of this PR.

Reply:
Ticket created: https://broadworkbench.atlassian.net/browse/IA-3166

Reply:
Even in the new Workspace column, sorting isn't working properly between runtimes and apps. I think that is expected and will be covered by IA-3166, but just wanted to confirm.

![image](https://user-images.githubusercontent.com/484484/149571143-af9aa84d-7de3-43c1-ba34-9fa13a3f7707.png)

Reply:
That is correct.